steward_proto/proto/aave_v2_stablecoin.proto

```diff
@@ -16,29 +16,31 @@ message AaveV2Stablecoin {
     // The function you wish to execute on the target cellar
     oneof function {
         // Represents function `accruePlatformFees()`
-        AccrueFees accrue_fees = 1;
+        Accrue accrue = 1;
```

Comment:
Breaking changes to the proto means this should probably be in the `steward.v2` package.